tests/unit/Statement/StatementListTest.php

```diff
@@ -275,7 +275,7 @@ public function testRemoveStatementsWithGuid_singleStatementRemoved() {
 		$this->assertEquals( $statements, $list->toArray() );
 	}
 
-	public function testRemoveStatementsWithGuid_multipleStatementsRemoved() {
+	public function testGivenGuidOfMulitpleStatements_multipleStatementsAreRemoved() {
```

Comment:
Do you know about this? http://dan.hersam.com/2005/01/27/reading-jumbled-letters/